commit 21ceeaabce80105976713ccada683a7ff7302370 (refs/changes/87/5687/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-01T13:11:15-08:00 (7 months ago)
    
    TRITON-1274 spurious test-amon failures in nightly: hit 180s timeout on maintenances.test.js

diff --git a/test/runtests b/test/runtests
index fcb0979..e257a92 100755
--- a/test/runtests
+++ b/test/runtests
@@ -179,7 +179,7 @@ if [[ -n "$opt_test_pattern" ]]; then
 fi
 # maintenances.test.js needs longer timeout
 if [[ -n "$test_files" ]]; then
-    PATH=$NODE_INSTALL/bin:$PATH TAP=1 $TAP --timeout 180 $test_files \
+    PATH=$NODE_INSTALL/bin:$PATH TAP=1 $TAP --timeout 3600 $test_files \
         | tee $OUTPUT_DIR/amon-relay.tap
 fi
 
